### PR TITLE
Baker summer survey

### PR DESCRIPTION
**Pull request by bakerfranke**
This PR adds a question panel to make use of the `is_friday_institute` variable.  Questions were added based on input/feedback from Friday Institute Staff.

@molly-moen please review.  Since this will be merged into the live survey, we want to take special care that it won't accidentally show up for non-friday-institute users.  Also need to make sure that the is_friday_institute variable is working properly.

### CODE REVIEW

File: dashboard/config/foorm/forms/surveys/pd/summer_workshop_post_survey.0.json [hunk 1]
```diff
@@ -196,6 +196,154 @@
...
+         "text": "Don't Know/Unsure"
+        }
+       ],
+       "hasOther": true,
+       "otherText": "Other"
+      },
+      {
+       "type": "matrix",
+       "name": "friday_institute_sync_tools",
+       "title": "Thinking about the video conferencing portions of the workshop, how much do you agree with the following?  ",
```
Comment (molly-moen) [inline dashboard/config/foorm/forms/surveys/pd/summer_workshop_post_survey.0.json]: remove extra spaces at the end of the sentence?

File: dashboard/config/foorm/forms/surveys/pd/summer_workshop_post_survey.0.json [hunk 2]
```diff
@@ -196,6 +196,154 @@
...
+         "text": "Agree"
+        },
+        {
+         "value": "7",
+         "text": "Strongly Agree"
+        }
+       ],
+       "rows": [
+        {
+         "value": "moodle_worked_well_for_all",
```
Comment (molly-moen) [inline dashboard/config/foorm/forms/surveys/pd/summer_workshop_post_survey.0.json:327]: should the value be moodle or place?

Comment (bakerfranke) [reply dashboard/config/foorm/forms/surveys/pd/summer_workshop_post_survey.0.json:327]: doesn't matter.  public facing name is PLACE.  But's a moodle site.  leave.

File: dashboard/config/foorm/forms/surveys/pd/summer_workshop_post_survey.0.json (hunk 1 above)
Comment (bakerfranke) [reply dashboard/config/foorm/forms/surveys/pd/summer_workshop_post_survey.0.json]: fixed

**Review by molly-moen — APPROVED** [review]

**Pull request merged**